curriculum/challenges/english/25-front-end-development/quiz-css-relative-and-absolute-units/66ed9009f45ce3ece4053eb7.md

```diff
@@ -413,23 +413,23 @@ Adds `10%` of the parent's width to `100px`
 
 #### --text--
 
-In CSS, how does the `pt` unit compare to the `px` unit on a typical screen with 96 DPI?
+What does setting an element's width to auto do in CSS?
```

Comment:
It's not a bad question, but `auto` is not mentioned in the topics for this quiz. WHat does @naomi-lgbt say?

Reply:
@naomi-lgbt  is it OK..??

Reply:
Should be fine.

Reply:
When it will be merged...??

Reply:
The rules below requires that there are no unresolved conversations and that the requests of changes are satisfied